mkosi/burn.py

```diff
@@ -0,0 +1,44 @@
+# SPDX-License-Identifier: LGPL-2.1+
+
+import os
+import sys
+
+from mkosi.config import (
+    MkosiArgs,
+    MkosiConfig,
+    OutputFormat,
+)
```

Comment:
The complaint's unclear, because it's sorted, but I just run isort on it and it basically just complains about line lenght
```suggestion
from mkosi.config import MkosiArgs, MkosiConfig, OutputFormat
```